aten/src/ATen/native/sparse/cuda/SparseCUDABlas.cu

```diff
@@ -71,6 +80,119 @@ cusparseOperation_t convertTransToCusparseOperation(char trans) {
   }
 }
 
+#if !defined(MSC_VER) && defined(__CUDACC__) && CUSPARSE_VERSION >= 10301 // CUDA release >= 10.2 and not windows
+
+template<typename T> 
+void csrmm2(
+  char transa, char transb, 
+  int64_t m, int64_t n, int64_t k, int64_t nnz, 
+  T alpha, T *csrvala, int *csrrowptra, int *csrcolinda, 
+  T *b, int64_t ldb, T beta, T *c, int64_t ldc)
+{
+  static_assert(std::is_same<float, T>::value || std::is_same<double, T>::value); 
+  constexpr auto cusparse_value_type = std::is_same<float, T>::value ? CUDA_R_32F : CUDA_R_64F; 
+
+  if (csrvala == nullptr || b == nullptr || c == nullptr) return; 
+
+  cusparseOperation_t opa = convertTransToCusparseOperation(transa);
+  cusparseOperation_t opb = convertTransToCusparseOperation(transb);
+
+  // cusparseSpMM actually supports int64_t. 
+  // In order to support int64 here, index pointers csrrowptra, csrcolinda have to be passed as int64_t. 
+  TORCH_CHECK((m <= INT_MAX) && (n <= INT_MAX) && (k <= INT_MAX) && (nnz <= INT_MAX) && (ldb <= INT_MAX) && (ldc <= INT_MAX),
+    "By now, cusparseSpMM only supports m, n, k, nnz, ldb, ldc with the bound [val] <= ", INT_MAX, ".", 
```

Comment:
"At the moment" would be more idiomatic

Reply:
And then I like to say, "If you need this, please file an issue on GitHub"

Reply:
fixed